crates/cli-support/src/js/binding.rs

```diff
@@ -564,6 +566,8 @@ fn instruction(js: &mut JsBuilder, instr: &Instruction, log_error: &mut bool) ->
                     writeln!(js.prelude, "{name} = {arg};").unwrap();
                     args.push(name);
                 }
+                // add alignment
+                args.push(String::from("4"));
```

Comment:
This should probably be added as a field of the instruction instead of hardcoded.

Reply:
Done.